domains/crischutu07.json

```diff
@@ -1,9 +1,14 @@
 {
-    "owner": {
-        "username": "crischutu07",
-        "email": "sondolavnxbox.omlet@gmail.com"
-    },
-    "record": {
-        "CNAME": "crischutu07.github.io"
-    }
+  "owner": {
+    "username": "crischutu07",
+    "email": "crischutu07owo@gmail.com"
+  },
+  "record": {
+    "CNAME": "crischutu07.github.io",
+    "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
You cant have other records with `CNAME` 
You can use `A` records instead of `CNAME` with four ips pointing to your website

Reply:
> Isn't `CNAME` enough for pointing to my GitHub Pages? I think you're confused

CNAME is enough for pointing to github pages, but you can't use CNAME with other records (MX, TXT etc), which is what you're doing here.

Reply:
> Isn't `CNAME` enough for pointing to my GitHub Pages? I think you're confused

you are confused. with CNAME you cant use it with any other records (like MX and TXT), but with A record (which github pages does support) will let you use MX and TXT